tools/checklocks/facts.go

```diff
@@ -686,3 +687,18 @@ func (pc *passContext) exportFunctionFacts(d *ast.FuncDecl) {
 		pc.pass.ExportObjectFact(funcObj, &lff)
 	}
 }
+
+// PKfPerfFacts stores revelant analysis stats for processing in benchlocks.
+type PkgPerfFacts struct {
+	// ErrorSiteCount records the total number of errors reported at a token.Pos
+	ErrorSiteCount map[token.Pos]int
+
+	// BasicBlocksVisits stores the average number of visits made per basic block
+	// in the control flow graph of a function.
+	BasicBlockVisits map[string]int
```

Comment:
Didn't end up actually using this stat, as every basic block is only visited once in the current implementation.

Reply:
Lets keep it. IIUC we believe we need it for defers. We will also need this for comparing to the path sensitive version.